### ISSUE TITLE
Busybox: psad_fw_read.pid does not exist 

### ISSUE THREAD

**Issue by itoffshore** [mrash/psad#31]
Under Alpine Linux / Busybox (without `systemd` / `journalctl`) in `psad 2.4.3` I see:

```
root@alpine [/etc/psad]# psad -S
[+] psadwatchd (pid: 20208)
[-] psad: pid file /var/run/psad/psad_fw_read.pid does not exist for psad_fw_read on ..............
[+] psad (pid: 20206)
root@alpine [/etc/psad]# ls /var/run/psad
.  ..  auto_ipt.sock  psad.cmd  psad.pid  psadwatchd.pid
```

`psad -K` & `psad -R` both work but also complain that `psad_fw_read.pid does not exist`.

I can see `ip's` are still blocked & system stats can still be viewed with `psad -A`. These warning messages seem to be false positives on systems without `systemd`.

**Comment by mrash** [comment]
Thanks for mentioning this - it is a status message that can be ignored. I'll quiet this msg down for non-journalcrtl systems in the next release.

**Comment by mrash** [comment]
Fixed in d7a40604.
